src/index.js

```diff
@@ -23,12 +22,14 @@ if (module.hot) {
   console.info('✅  Server-side HMR Enabled!'); // eslint-disable-line no-console
 }
 
-export default express()
-  .use((req, res) => app.handle(req, res))
```

Comment:
Så det er i bunn denne som har en egen listen, som vi legger på i etterkant?

Reply:
Ja vi kaller på `express().listen(...)` for å starte å lytte (og det gjør vi forsåvidt fortsatt bare jeg brukte den på `app` for jeg syns det var penere.
Men på vercel nå så virker det som de importerer det eksporterte objektet fra index-fila og gjør noe magi for at det skal fungere.
Så når vi lytter selv så ender vi med å lytte 2 ganger.